### ISSUE TITLE
implement vdom metrics

### ISSUE THREAD

**Issue by mbdraks** [mbdraks/fortinet-zabbix#14]

**Comment by mbdraks** [comment]
```
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdInfo.fgVdNumber|1.3.6.1.4.1.12356.101.3.1.1
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdInfo.fgVdMaxVdoms|1.3.6.1.4.1.12356.101.3.1.2
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdInfo.fgVdEnabled|1.3.6.1.4.1.12356.101.3.1.3
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntIndex|1.3.6.1.4.1.12356.101.3.2.1.1.1
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntName|1.3.6.1.4.1.12356.101.3.2.1.1.2
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntOpMode|1.3.6.1.4.1.12356.101.3.2.1.1.3
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntHaState|1.3.6.1.4.1.12356.101.3.2.1.1.4
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntCpuUsage|1.3.6.1.4.1.12356.101.3.2.1.1.5
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntMemUsage|1.3.6.1.4.1.12356.101.3.2.1.1.6
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntSesCount|1.3.6.1.4.1.12356.101.3.2.1.1.7
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntSesRate|1.3.6.1.4.1.12356.101.3.2.1.1.8
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTpTable.fgVdTpEntry.fgVdTpMgmtAddrType|1.3.6.1.4.1.12356.101.3.2.2.1.1
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTpTable.fgVdTpEntry.fgVdTpMgmtAddr|1.3.6.1.4.1.12356.101.3.2.2.1.2
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTpTable.fgVdTpEntry.fgVdTpMgmtMask|1.3.6.1.4.1.12356.101.3.2.2.1.3
```

**Comment by mbdraks** [comment]
```
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntIndex|1.3.6.1.4.1.12356.101.3.2.1.1.1
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTable.fgVdEntry.fgVdEntName|1.3.6.1.4.1.12356.101.3.2.1.1.2
```

Implemented on #31

**Comment by mbdraks** [comment]
sample data

```
# snmpwalk -v2c -c comm host fgVirtualDomain
FORTINET-FORTIGATE-MIB::fgVdNumber.0 = INTEGER: 1
FORTINET-FORTIGATE-MIB::fgVdMaxVdoms.0 = INTEGER: 10
FORTINET-FORTIGATE-MIB::fgVdEnabled.0 = INTEGER: disabled(1)
FORTINET-FORTIGATE-MIB::fgVdEntIndex.1 = INTEGER: 1
FORTINET-FORTIGATE-MIB::fgVdEntName.1 = STRING: root
FORTINET-FORTIGATE-MIB::fgVdEntOpMode.1 = INTEGER: nat(1)
FORTINET-FORTIGATE-MIB::fgVdEntHaState.1 = INTEGER: master(1)
FORTINET-FORTIGATE-MIB::fgVdEntCpuUsage.1 = Gauge32: 0
FORTINET-FORTIGATE-MIB::fgVdEntMemUsage.1 = Gauge32: 25
FORTINET-FORTIGATE-MIB::fgVdEntSesCount.1 = Gauge32: 3994
FORTINET-FORTIGATE-MIB::fgVdEntSesRate.1 = Gauge32: 36 Sessions Per Second
```

**Comment by mbdraks** [comment]
the following OIDs will not be implemented for now

```
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTpTable.fgVdTpEntry.fgVdTpMgmtAddrType|1.3.6.1.4.1.12356.101.3.2.2.1.1
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTpTable.fgVdTpEntry.fgVdTpMgmtAddr|1.3.6.1.4.1.12356.101.3.2.2.1.2
FORTINET-FORTIGATE-MIB:fortinet.fnFortiGateMib.fgVirtualDomain.fgVdTables.fgVdTpTable.fgVdTpEntry.fgVdTpMgmtMask|1.3.6.1.4.1.12356.101.3.2.2.1.3
```

**Comment by mbdraks** [comment]
Closed by #42
